src/rest/orders.rs

```diff
@@ -49,7 +49,7 @@ impl From<OrderLinkId> for ActiveOrderId {
     }
 }
 
-#[derive(Default)]
+#[derive(Debug, Default)]
```

Comment:
Can we leave this for another PR? I think this PR should focus -as the title suggests- on adding ? support and implementing Display for errors.

Reply:
fixed